gradle.properties

```diff
@@ -1,44 +1,59 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=swan-lake-preview6-SNAPSHOT
-ballerinaVersion=2.0.0-Preview5
-shortVersion=slp5
+ballerinaVersion=2.0.0-Preview6-SNAPSHOT
+shortVersion=slp6
 
 # Following have already been released to ballerina central
-stdlibSocketVersion=0.6.9
-stdlibEncodingVersion=1.0.3
-stdlibJwtVersion=1.0.3
-stdlibWebsubVersion=1.0.3
-stdlibRabbitmqVersion=1.0.4
-stdlibNatsVersion=1.0.2
-
-stdlibXsltVersion=1.0.2
-stdlibFileVersion=0.5.2
-stdlibTimeVersion=1.0.2
-stdlibFilepathVersion=0.7.2
-stdlibTaskVersion=1.1.1
-stdlibCacheVersion=2.0.1
-stdlibSystemVersion=0.6.1
-stdlibRuntimeVersion=0.5.1
-stdlibConfigVersion=1.0.1
-stdlibSqlVersion=0.5.1
-stdlibJdbcVersion=0.5.1
-stdlibMySqlVersion=0.6.1
-stdlibKafkaVersion=2.0.1
-stdlibMathVersion=1.0.1
-stdlibJsonUtilsVersion=1.0.1
-stdlibXmlUtilsVersion=1.0.1
-stdlibLogVersion=1.0.1
-stdlibReflectVersion=0.5.1
-stdlibIoVersion=0.5.1
-stdlibStringutilsVersion=0.5.1
-stdlibAuthVersion=1.0.1
-stdlibOAuth2Version=1.0.1
-stdlibLdapVersion=1.0.1
-stdlibCryptoVersion=1.0.1
-stdlibJavaArraysVersion=0.9.1
-stdlibMimeVersion=1.0.1
-stdlibEmailVersion=1.0.1
-stdlibFtpVersion=1.0.0
-stdlibHttpVersion=1.0.1
-stdlibGrpcVersion=0.7.1
+stdlibSocketVersion=0.7.0-SNAPSHOT
+stdlibEncodingVersion=1.0.4-SNAPSHOT
+stdlibJwtVersion=1.0.4-SNAPSHOT
+stdlibWebsubVersion=1.0.4-SNAPSHOT
```

Comment:
This should be `1.0.5-SNAPSHOT`

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/923/commits/41dda352ffce009da3fd47b22dbb317170c21ae8